docs/_advanced/lazy_listener.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Lazy listeners (FaaS)
+lang: en
+slug: lazy-listeners
+order: 9
+---
+
+<div class="section-content">
+⚠️ Lazy listener functions are a beta feature to make it easier to deploy Bolt for Python apps to FaaS environments. As the feature is developed, Bolt for Python's API is subject to change.
+
+Typically you'd call `ack()` as the first step of your listener functions. Calling `ack()` tells Slack that you've received the event and are handling it in within reasonable amount of time (3 seconds).
+
+However, apps running on FaaS or similar runtimes that don't allow you to run threads or processes after returning an HTTP response cannot follow this pattern. Instead, you should set the `process_before_response` flag to `true`. This allows you to call ack() and handle the event in seperate threads or processes. For events that do not require acknowledgement, you can create the listener as you normally would.
+
+Rather than acting as a decorator, lazy listeners take two keyword args:
+* `ack: Callable`: Responsible for calling `ack()`
+* `lazy: List[Callable]`: Responsible for handling any time-consuming processes related to the event. The lazy function does not have access to `ack()`.
+</div>
+
+```python
+app.command("/start-process")(
+    # ack() is still called within 3 seconds
+    ack=respond_to_slack_within_3_seconds,
+    # Lazy function is responsible for processing the event
+    lazy=[run_long_process]
+)
+```
+
+<details class="secondary-wrapper">
+<summary class="section-head" markdown="0">
+<h4 class="section-head">Example with AWS Lambda</h4>
+</summary>
+
+<div class="secondary-content" markdown="0">
+This example deploys the code to [AWS Lambda](https://aws.amazon.com/lambda/), but lazy listener functions work well in most FaaS enviornments. There are more examples within the [`sample` folder](https://github.com/slackapi/bolt-python/tree/main/adapter).
```

Comment:
Other platforms are not yet supported. This feature works with built-in adapters, all non-FaaS adapters, and AWS Lambda at this point.
```suggestion
This example deploys the code to [AWS Lambda](https://aws.amazon.com/lambda/). There are more examples within the [`sample` folder](https://github.com/slackapi/bolt-python/tree/main/adapter).
```